core/README.md

```diff
@@ -1,4 +1,4 @@
-# libtock-core
+# libtock\_core
```

Comment:
Why the `\`?

Reply:
It's to escape the `_`, which is otherwise interpreted by Markdown as an italics tag.

Reply:
Ah ok. Make sense.